CONTRIBUTING.md

```diff
@@ -84,3 +84,12 @@ You can do so as:
 $ pytest
 ```
 
+### Additional system requirements for running the tests:
+
+- `docker`
+- `conda`: for example `Miniconda`, follow the steps described in this [link](https://docs.conda.io/projects/conda/en/latest/user-guide/).
+- Models artifacts:
+For first time, install `gsutil` and download the models from Seldon's Google Storage bucket:
+```bash
+$ sudo make tests/testdata
```

Comment:
Is `sudo` needed to fetch the test models?

Reply:
yes at least on my local machine to be able to store the files from gs.

Reply:
I think it might be just an isolated use case and will remove `sudo`